src/asm_ostream.cpp

```diff
@@ -357,7 +357,8 @@ auto get_labels(const InstructionSeq& insts) {
     return pc_of_label;
 }
 
-void print(const InstructionSeq& insts, std::ostream& out, std::optional<const label_t> label_to_print) {
+void print(const InstructionSeq& insts, std::ostream& out, std::optional<const label_t> label_to_print,
+           std::function<void(std::ostream&, const btf_line_info_t&)> line_info_formatter) {
```

Comment:
The new parameter is unused. What am I missing?

Reply:
Simplified this to just accept a bool.